app/controllers/users_controller.rb

```diff
@@ -32,7 +32,7 @@ def join_team
   def leave_team
     is_current_captain = @team.team_captain&.id.eql?(params[:user_id].to_i)
 
-    if is_current_captain && @team.users.size > 1
+    if is_current_captain && @team.users.count > 1
```

Comment:
Wouldn't hurt to add a note here as to why we are using count instead of size so the next person doesn't try to "optimize" it.